pkg/controller/mustgatherservice/mustgatherservice.go

```diff
@@ -392,32 +392,37 @@ func (r *ReconcileMustGatherService) updateMustGatherServiceIngress(instance *op
 
 }
 
-func (r *ReconcileMustGatherService) desiredMustGatherServiceIngress(instance *operatorv1alpha1.MustGatherService) *extensionsv1.Ingress {
+func (r *ReconcileMustGatherService) desiredMustGatherServiceIngress(instance *operatorv1alpha1.MustGatherService) *networkingv1.Ingress {
 	appName := mustGatherResourceName
 	labels := labelsForMustGatherService(appName, instance.Name)
 	annotations := annotationsForMustGatherServiceIngress()
+	pathType := networkingv1.PathType("ImplementationSpecific")
 
 	reqLogger := log.WithValues("MustGatherService.Namespace", instance.Namespace, "MustGatherService.Name", instance.Name)
 	reqLogger.Info("Building MustGatherService Ingress", "Ingress.Namespace", instance.Namespace, "Ingress.Name", appName)
 
-	ing := &extensionsv1.Ingress{
+	ing := &networkingv1.Ingress{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        appName,
 			Namespace:   instance.Namespace,
 			Labels:      labels,
 			Annotations: annotations,
 		},
-		Spec: extensionsv1.IngressSpec{
-			Rules: []extensionsv1.IngressRule{
+		Spec: networkingv1.IngressSpec{
+			Rules: []networkingv1.IngressRule{
 				{
-					IngressRuleValue: extensionsv1.IngressRuleValue{
-						HTTP: &extensionsv1.HTTPIngressRuleValue{
-							Paths: []extensionsv1.HTTPIngressPath{
+					IngressRuleValue: networkingv1.IngressRuleValue{
+						HTTP: &networkingv1.HTTPIngressRuleValue{
+							Paths: []networkingv1.HTTPIngressPath{
 								{
-									Path: "/must-gather/",
-									Backend: extensionsv1.IngressBackend{
-										ServiceName: appName,
-										ServicePort: intstr.IntOrString{Type: intstr.Int, IntVal: 6967}},
+									Path:     "/must-gather/",
+									PathType: &pathType,
+									Backend: networkingv1.IngressBackend{
+										Service: &networkingv1.IngressServiceBackend{
+											Name: appName,
+											Port: networkingv1.ServiceBackendPort{Number: 6967},
```

Comment:
Please change Number: `6967` into constant variable

Reply:
Introduced constant file to address these values.